plugin/src/main/java/org/jenkinsci/plugins/structs/SymbolLookup.java

```diff
@@ -152,5 +158,44 @@ public static SymbolLookup get() {
         return j.getInjector().getInstance(SymbolLookup.class);
     }
 
+    /**
+     * Get the {@link Symbol} value(s) for the class of the given object, generally a {@link Descriptor}, if the annotation
+     * is present. If the object is in fact a {@link Describable}, we'll use its {@link Descriptor} class instead.
+     *
+     * @param o An object
+     * @return The {@link Symbol} annotation value(s) for the class (generally a {@link Descriptor} that object represents,
+     * or an empty {@link Set} if the annotation is not present.
+     */
+    @Nonnull public static Set<String> getSymbolValue(@Nonnull Object o) {
+        if (o instanceof Describable) {
+            return getSymbolValue(((Describable) o).getDescriptor().getClass());
+        } else {
+            return getSymbolValue(o.getClass());
+        }
+    }
+
+    /**
+     * Get the {@link Symbol} value(s) for the given class, if the annotation is present. Unlike {@link #getSymbolValue(Object)},
+     * this will not get the {@link Descriptor} for {@link Describable} classes.
+     *
+     * @param c A class.
+     * @return The {@link Symbol} annotation value(s) for the given class, or an empty {@link Set} if the annotation is not present.
+     */
+    @SuppressFBWarnings(value = "NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE",
+            justification = "Jenkins.getInstance() can return null in theory.")
```

Comment:
The proper fix (pending some Jenkins 2.x version where this method is `@Nonnull`) is to assign `Jenkins.getInstance()` to a variable, check it for null, then proceed.